stand/lua/config.lua.8

```diff
@@ -24,12 +24,12 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd December 17, 2020
+.Dd March 29, 2025
 .Dt CONFIG.LUA 8
 .Os
 .Sh NAME
 .Nm config.lua
-.Nd FreeBSD config module
+.Nd bootloader configuration module
```

Comment:
config, though, is the name of the Lua module, which is important to know to use it.

Reply:
That's in the name section sir. It renders:
```
config.lua(8) - bootloader configuration module
```